examples/telemetry_takeoff_and_land.py

```diff
@@ -46,14 +46,18 @@ async def run():
             print("-- Global position state is good enough for flying.")
             break
 
+    # Take the drone little bit higher during takeoff
+    print("-- Setting takeoff altitude")
+    await drone.param.set_param_float('MIS_TAKEOFF_ALT', 10.0)
```

Comment:
Actually, you can just use `action.set_takeoff_altitude(10)` instead. That's the more canonical way to do that.